store/index.js

```diff
@@ -143,6 +143,12 @@ function getDefaultState() {
 
             Label: "period of time defined according to the ISO8601 standard",
             TermURL: "nb:FromISO8601"
+        },
+
+        range: {
+
+            Label: "Age defined as a range of two integers",
+            TermURL: "nb:FromRanged"
```

Comment:
Hey @surchs, I wanted to quickly double check how the annotation was defined here to match the corresponding CLI [PR](https://github.com/neurobagel/bagel-cli/pull/437):

- I think we decided in the [issue](https://github.com/neurobagel/legacy_annotation_tool/issues/862#issuecomment-2683465926) to call this `nb:FromRange` to match our other format names!
- Do you think the integer constraint is important? For the CLI at least, numbers on either side of `-` are converted to float before the average is calculated (this is also consistent with the legacy behaviour) so whether the min/max are integers or floats doesn't really matter 🤷

Reply:
aha, good point, thanks. I went off the issue description - seems we didn't update to `nb:FromRange` there. 

Yeah, int is probably not very important, we can go with Float. I'll make those changes